src/odemis/gui/cont/tabs.py

```diff
@@ -3691,6 +3678,7 @@ def _onManualFocus(self, event):
             f.result()  # TODO: don't block the GUI => make it part of the manual focus??
 
             # Enable slider for manual alignment of spectrograph focus
+            # TODO include in xml as additional arg active_color
```

Comment:
Yes, indeed... but then you'll have to "bite the bullet", and extend the xh_delmic file first ;-)